.travis.rosinstall

```diff
@@ -1,7 +1,7 @@
 - git:
     uri: https://github.com/jsk-ros-pkg/jsk_common.git
     local-name: jsk-ros-pkg/jsk_common
-    version: master
+    version: 2.0.16
```

Comment:
python-termcolor is not released on saucy and will not included deb
dependency, but if you wait 1 more week, we can release new version
including python-termcolor due to terminating saucy support for indigo (
http://lists.ros.org/pipermail/ros-release/2016-June/004879.html)
$ rosdep resolve --os=ubuntu:saucy python-termcolor
#pip
termcolor


--
◉ Kei Okada


On Wed, Jun 22, 2016 at 4:31 PM, Kentaro Wada <notifications@github.com>
wrote:

> In .travis.rosinstall
> <https://github.com/jsk-ros-pkg/jsk_recognition/pull/1750#discussion_r68006114>
> :
>
> > @@ -1,7 +1,7 @@
> >  - git:
> >      uri: https://github.com/jsk-ros-pkg/jsk_common.git
> >      local-name: jsk-ros-pkg/jsk_common
> > -    version: master
> > +    version: 2.0.16
>
> It needs python-termcolor.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/jsk-ros-pkg/jsk_recognition/pull/1750/files/81315abc27562253ac8d9191f7c2f17d2545408e#r68006114>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAeG3LjFWCU6tI69qEuzo-VuvzUVFMr_ks5qOOS8gaJpZM4I3fwk>
> .
>